tests/unittests/graphTest.cpp

```diff
@@ -359,6 +360,11 @@ TEST(Graph, cloneTest2) {
 
   auto *newF = F->clone("new_main");
   newF->verify();
+  F->dump();
+  F->dumpDAG();
```

Comment:
Do we really need `dumpDAG` or may be just `dump` is enough? I'm asking because `dumpDAG` creates a new file each time it runs. If we can avoid it, we should not call it.

Reply:
dump is enough. Switching to this. The reason I went with both was because the other tests use both.

Reply:
Slightly troll-ish suggestion: `dumpDAG("/dev/stdout")`  :-D